cli/deno_dir.rs

```diff
@@ -661,27 +629,28 @@ fn fetch_remote_source_async(
       deno_dir.clone(),
       None,
       None,
-      module_name.clone(),
+      module_url.clone(),
       filepath.clone(),
     ),
     |(
       dir,
       mut maybe_initial_module_name,
       mut maybe_initial_filepath,
-      module_name,
+      module_url,
       filepath,
     )| {
-      let url = module_name.parse::<http::uri::Uri>().unwrap();
+      // TODO: there must be better way to do this conversion
+      let url = module_url.to_string().parse::<http::uri::Uri>().unwrap();
```

Comment:
Can you add 
```rust
impl From<url::Url> for http::uri::Uri {
  fn from(url: &url::Url) -> http::uri::Uri {
    // TODO(bartlomieju) Is there a better way to do this conversion?
    module_url.to_string().parse::<http::uri::Uri>().unwrap()
  }
}
```

Reply:
It errors:
```
error[E0117]: only traits defined in the current crate can be implemented for arbitrary types
```
AFAIK one can't implement traits like this (from Rustbook):

> But we can’t implement external traits on external types. For example, we can’t implement the Display trait on Vec<T> within our aggregator crate, because Display and Vec<T> are defined in the standard library and aren’t local to our aggregator crate. This restriction is part of a property of programs called coherence, and more specifically the orphan rule, so named because the parent type is not present.

Reply:
oops! There's a way to work around this by defining a new trait that inherits from Uri, but maybe it's better to just move this into a function 
```rust
fn url_into_uri((url: &url::Url) -> http::uri::Uri {
  // TODO(bartlomieju) Is there a better way to do this conversion?
  module_url.to_string().parse::<http::uri::Uri>().unwrap()
}
```